docs/Participate/Setup-Project.md

```diff
@@ -38,7 +38,32 @@ In IntelliJ click on `Git` in the left upper corner and then `Manage Remotes...`
 In the new window you already see a remote called `origin`. This remote is your fork of BetonQuest.
 Now add a new repository with the name `upstream` and the url `https://github.com/BetonQuest/BetonQuest.git`.
 
-##Building the Plugin jar
+## IntelliJ settings
+Formatting for .md (Markdown) files can break some features of
+<a href="https://squidfunk.github.io/mkdocs-material/" target="_blank">Material for MkDocs</a>, so we disable it for these files.
```

Comment:
See https://github.com/BetonQuest/BetonQuest/pull/1935#pullrequestreview-1037906980 for detailed explanation why I think opening links in a new tab by default is a bad idea.

```suggestion
[Material for MkDocs](https://squidfunk.github.io/mkdocs-material/), so we disable it for these files.
```